openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -504,7 +504,13 @@ def process(self, instance):
 
         self.payload_skeleton["JobInfo"]["Comment"] = comment
         self.payload_skeleton["PluginInfo"]["RenderLayer"] = renderlayer
-
+        self.payload_skeleton["PluginInfo"]["RenderSetupIncludeLights"] = (
+            context.data["project_settings"].get(
+                "maya", {}).get(
+                "create", {}).get(
+                "CreateRender", {}).get(
+                "enable_all_lights", {})
+        )
```

Comment:
This is the issue that others are reporting where the value in Deadline is incorrect - it'll always use the project settings value instead of the value on the instance. 

Instead it should likely be `instance.data.get("RenderSetupIncludeLights")` like [here](https://github.com/pypeclub/OpenPype/pull/3038/files#diff-b331d4b66a42158b9abaf95dab978dcc14b362e492941dc40c40e3ce82e9aa31R252).

Note that this instance data key does seem to deviate from most others I have seen being stored as pyblish instance data - I don't recall them starting with upper case characters before - I'd have expected `renderSetupIncludeLights`?

Reply:
agree, all keys should start with lower letter.

Reply:
Thanks guys, will address both points right now.